lib/fs.js

```diff
@@ -1615,6 +1615,11 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (typeof options === 'string')
```

Comment:
Yes, I would like to fix the direction. 

Most functions in `fs` have same behavior... That is historical background. 
If we correct them, some developers should fix the option from `string` to `object`.
That change cause `major` version up...